### PR TITLE
dxScheduler - add Vue and React examples to customizeDateNavigatorText article

### PR DESCRIPTION
**Pull request by andrewmakarov**

### CODE REVIEW

**Review by pavel-2020 — APPROVED** [review]

File: api-reference/10 UI Widgets/dxScheduler/1 Configuration/customizeDateNavigatorText.md [hunk 1]
```diff
@@ -85,10 +85,84 @@ In the following code, the **customizeDateNavigatorText** function is used to sh
         // ...
     })
 
-    <!--HTML-->
-    <dx-scheduler ...
-        [(currentView)]="currentView"
-        [customizeDateNavigatorText]="customizeDateNavigatorText">
-    </dx-scheduler>
+##### Vue
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxScheduler/1 Configuration/customizeDateNavigatorText.md:94]: Why the lines above were removed?

File: api-reference/10 UI Widgets/dxScheduler/1 Configuration/customizeDateNavigatorText.md [hunk 2]
```diff
@@ -85,10 +85,84 @@ In the following code, the **customizeDateNavigatorText** function is used to sh
...
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxScheduler ...
+            :customize-date-navigator-text="customizeDateNavigatorText">
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxScheduler/1 Configuration/customizeDateNavigatorText.md:100]:
```suggestion
            v-model:current-view="currentView"
            :customize-date-navigator-text="customizeDateNavigatorText">
```

File: api-reference/10 UI Widgets/dxScheduler/1 Configuration/customizeDateNavigatorText.md [hunk 3]
```diff
@@ -85,10 +85,84 @@ In the following code, the **customizeDateNavigatorText** function is used to sh
...
+            const formattedEndDate = e.endDate.toLocaleString('en', formatOptions);
+            if(this.currentView === 'day' | 'timelineDay') 
+                return formattedStartDate;
+            if(this.currentView === 'month' ) 
+                return e.startDate.toLocaleString('en', { year: 'numeric', month: 'numeric' });
+            return formattedStartDate + ' - ' + formattedEndDate;
+        }, [currentView]);
+
+        return (
+            <Scheduler ...
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxScheduler/1 Configuration/customizeDateNavigatorText.md:174]: We need to handle `currentView` changes for this to work.

File: api-reference/10 UI Widgets/dxScheduler/1 Configuration/customizeDateNavigatorText.md [hunk 4]
```diff
@@ -91,4 +91,94 @@ In the following code, the **customizeDateNavigatorText** function is used to sh
...
+
+        const customizeDateNavigatorText = useCallback((e) => {
+            const formatOptions = { 
+                year: 'numeric', 
+                month: 'numeric', 
+                day: 'numeric' 
+            };
+            const formattedStartDate = e.startDate.toLocaleString('en', formatOptions);
+            const formattedEndDate = e.endDate.toLocaleString('en', formatOptions);
+            if(this.currentView === 'day' | 'timelineDay')
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxScheduler/1 Configuration/customizeDateNavigatorText.md]: Are you sure that `this` is required?

```suggestion
            if(currentView === 'day' | 'timelineDay') 
```

File: api-reference/10 UI Widgets/dxScheduler/1 Configuration/customizeDateNavigatorText.md [hunk 5]
```diff
@@ -91,4 +91,94 @@ In the following code, the **customizeDateNavigatorText** function is used to sh
...
+            const formatOptions = { 
+                year: 'numeric', 
+                month: 'numeric', 
+                day: 'numeric' 
+            };
+            const formattedStartDate = e.startDate.toLocaleString('en', formatOptions);
+            const formattedEndDate = e.endDate.toLocaleString('en', formatOptions);
+            if(this.currentView === 'day' | 'timelineDay') 
+                return formattedStartDate;
+            if(this.currentView === 'month' )
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxScheduler/1 Configuration/customizeDateNavigatorText.md]:
```suggestion
            if(currentView === 'month' ) 
```

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**